src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,19 @@
-const fizzBuzz = number => {};
+const fizzBuzz = number => {
+  const isDivisableBy3 = number % 3 === 0;
```

Comment:
I love how clearly you've written this out. Really easy for a human to read and great for learning. The smallest common multiple of both 5 and 3 is 15 so you can also use `number % 15 === 0` for the "FizzBuzz" option